src/rdkafka.h

```diff
@@ -9848,6 +9869,221 @@ RD_EXPORT void rd_kafka_DeleteAcls(rd_kafka_t *rk,
                                    const rd_kafka_AdminOptions_t *options,
                                    rd_kafka_queue_t *rkqu);
 
+/**
```

Comment:
Start a new section for ElectLeaders docs. The ongoing section is for DeleteAcls.